recipes/telluric/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "telluric" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "88c92f26b75d3600d4546a533292a13ceb38a37e93e627a232b72308075fd0a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - affine
+    - dsnparse
+    - fiona >=1.7.2,<2.*
+    - folium
+    - ipyleaflet
+    - pyproj
+    - shapely >=1.6.3,<2.*
+    - rasterio >=1.0a12
```

Comment:
I think you're going to need to put in a pull request on the `rasterio` feedstock to get the latest version up there. I'm not sure about the alpha status though.

Reply:
The package is there, under a different channel https://anaconda.org/conda-forge/rasterio/files?channel=dev

Reply:
I'm glad to see that rasterio 1.0 will be out in a couple of months :) Moving rasterio seems not worth it, and neither does moving telluric. Should I close this PR or just leave it open until the new stable package is out?